src/views/Header/HeaderBackButton.js

```diff
@@ -105,12 +105,13 @@ class HeaderBackButton extends React.PureComponent {
 
   render() {
     const { onPress, pressColorAndroid, title } = this.props;
+    const accessibilityLabel = title ? title : 'header-back';
```

Comment:
`header-back` is not a very good label for accessibility. can you change it to something like `Back` instead?